newdoc/newdoc/newdoc.py

```diff
@@ -7,22 +7,10 @@
 import argparse
 from string import Template
 
-NEWDOC_VERSION = "1.3.2"
-
-# Record whether we're running under Python 2 or 3
-PYVERSION = sys.version_info.major
+NEWDOC_VERSION = "1.5.0"
 
 # The configparser module is called ConfigParser in Python2
-if PYVERSION == 2:
-    import ConfigParser as cp
-else:
-    import configparser as cp
-
-# Force Python 2 to use the UTF-8 encoding. Otherwise, loading a template
-# containing Unicode characters fails.
-if PYVERSION == 2:
-    reload(sys)
-    sys.setdefaultencoding('utf8')
+import configparser as cp
```

Comment:
Not part of the changes, but you could easily import the three things instead of the module, I think.

Reply:
Do you mean `ConfigParser`, `MissingSectionHeaderError`, and `NoOptionError`? Sure, importing them directly could somewhat optimize the performance if I called the objects in a loop. But since they aren't in a performance-sensitive place, I prefer to keep the namespace to make it clear where the objects come from.